generate.sh

```diff
@@ -12,6 +12,8 @@ scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
 RED='\033[0;31m'
 NC='\033[0m'
 
+backend="cci"
```

Comment:
I feel like it would be best to group the option parameters together for readability/maintainability.  Consider pulling the others up here.